lib/graphql_devise/mutations/update_password_with_token.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module GraphqlDevise
+  module Mutations
+    class UpdatePasswordWithToken < Base
+      argument :password,              String, required: true
+      argument :password_confirmation, String, required: true
+      argument :reset_password_token,  String, required: true
+
+      field :credentials,
+            GraphqlDevise::Types::CredentialType,
+            null:        true,
+            description: 'Authentication credentials. Resource must be signed_in in order for credentials to be returned.'
```

Comment:
What about `Authentication credentials. Resource must be signed_in for credentials to be returned.` ?

Reply:
yes, makes more sense